src/components/MapboxMap/index.tsx

```diff
@@ -5,7 +5,7 @@ import ZoomControl from "mapbox-gl-controls/lib/zoom";
 
 import MapboxContext, {
   MapboxMapTransform,
-} from "../../contexts/mapboxContext";
+} from "../../contexts/MapboxContext";
```

Comment:
grrr I thought this might be a problem. Thanks for fixing.

I think this is borken for me because I ran `yarn link` and _then_ changed the case of this file, and MacOS' file system isn't case sensitive. So... the lesson is don't do that I guess